docs/examples/auto-instrumentation/README.rst

```diff
@@ -148,7 +147,7 @@ and run the following command instead:
 
 .. code:: sh
 
-    $ opentelemetry-instrument python server_uninstrumented.py
+    $ opentelemetry-instrument -e 'none' python server_uninstrumented.py
```

Comment:
I don't think this is the correct way to go about this.

By setting a none / invalid value, I think you are forcing the default of opentelemetry-python, which is the ConsoleSpanExporter. This should be explicitly set, rather than assuming that non-specified defaults will change.

There's a couple changes that need to be made, to make this happen:

1. add both span and metric console exporters to the [sdk setup.cfg](https://github.com/open-telemetry/opentelemetry-python/blob/master/opentelemetry-sdk/setup.cfg#L54), similar to https://github.com/open-telemetry/opentelemetry-python/blob/master/exporter/opentelemetry-exporter-otlp/setup.cfg#L57.
2. modify this statement to `-e 'console_span,console_metric'`.

Also this seems like an undesirable default to me (otlp), but it's specified in the spec. So [here's an issue](https://github.com/open-telemetry/opentelemetry-specification/issues/985) if you're interested in chiming in or following along.

Reply:
thanks @toumorokoshi . Infact i did those changes in setup.cfg and reverted back after i discovered the "none" option in the doc. I will go ahead and modify as per your suggestion.